sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala

```diff
@@ -118,7 +118,45 @@ object GenerateOrdering extends CodeGenerator[Seq[SortOrder], Ordering[InternalR
           }
       """
     }.mkString("\n")
-    comparisons
+
+    /*
```

Comment:
Thanks @kiszk for bringing this up! @ueshin @kiszk any comments please on do we want to expand `CodeGenerator#private splitExpressions` to:
 - add one more argument to specific how we want to "return" things?
 - add arguments to specific in which manner do we want to break expression? E.g. in the pr's case, we might want to break it according to the `numberOfComparisonsThreshold = 40` rather than on the string length